modin/pandas/dataframe.py

```diff
@@ -1523,6 +1523,9 @@ def pivot(self, index=None, columns=None, values=None):  # noqa: PR01, RT01, D20
         """
         Return reshaped ``DataFrame`` organized by given index / column values.
         """
+        # If values is None, we use all remaining columns not in index or columns.
+        if values is None:
+            values = [c for c in self.columns if c not in index and c not in columns]
```

Comment:
I wonder how this matches with PR title... seems like an unrelated change.

Reply:
yes, we should revert this change. Let's move it to https://github.com/modin-project/modin/issues/6146.